rhino-utils/README.md

```diff
@@ -99,17 +99,17 @@ cd path/to/your/code
 
 ## Using the Docker Push Script
 
-1. If you have not already done so, set up your ECR credentials with the AWS CLI. If you need assistance in setting these up, please follow the instructions in the `Setup` steps of [Pushing Generalized Compute Image Containers](https://docs.rhinohealth.com/hc/en-us/articles/6040656682269-Pushing-Generalized-Compute-Container-Images) guide. If your credentials have been updated and you need assistance updating them in your local config, please review this guide: [How can I setup or Change my ECR Credentials?](https://rhinohealth.zendesk.com/hc/en-us/articles/11383336127133)
-2. Find your ECR workgroup repository name (e.g. `rhino-gc-workgroup-my-workgroup`). If you need help finding your ECR workgroup repository name, please follow this guide [How can I find my ECR Workgroup Repository Name, ECR Access Key ID, ECR Secret Access Key?](https://rhinohealth.zendesk.com/hc/en-us/articles/9073591628189)
+1. If you have not already done so, authenticate to your cloud provider CLI. For AWS, set up your ECR credentials with the AWS CLI. If you need assistance in setting these up, please follow the instructions in the `Setup` steps of [Pushing Generalized Compute Image Containers](https://docs.rhinohealth.com/hc/en-us/articles/6040656682269-Pushing-Generalized-Compute-Container-Images) guide. If your credentials have been updated and you need assistance updating them in your local config, please review this guide: [How can I setup or Change my ECR Credentials?](https://rhinohealth.zendesk.com/hc/en-us/articles/11383336127133). For GCP, run `gcloud auth login` and follow the instructions.
+2. Find your workgroup repository name in the container image registry (e.g. `rhino-gc-workgroup-my-workgroup`). If you need help finding your workgroup repository name, please follow this guide [How can I find my ECR Workgroup Repository Name, ECR Access Key ID, ECR Secret Access Key?](https://docs.rhinohealth.com/hc/en-us/articles/12780382529309-How-can-I-find-my-SFTP-Server-Name-IP-Address-SFTP-Username-SFTP-Password-ECR-Workgroup-Repository-Name-ECR-Access-Key-ID-ECR-Secret-Access-Key)
```

Comment:
Can we make this less AWS-centric? I've filed an issue for Antoni to update the documentation as well.
```suggestion
2. Find your workgroup repository name in the container image registry (e.g. `rhino-gc-workgroup-my-workgroup`). If you need help finding your workgroup repository name, please follow this guide [How can I find my Workgroup Repository Name, Access Key ID, and Secret Access Key?](https://docs.rhinohealth.com/hc/en-us/articles/12780382529309-How-can-I-find-my-SFTP-Server-Name-IP-Address-SFTP-Username-SFTP-Password-ECR-Workgroup-Repository-Name-ECR-Access-Key-ID-ECR-Secret-Access-Key)
```